ryanair/ryanair.py

```diff
@@ -191,7 +206,9 @@ def _on_query_error(e):
     def _retryable_query(self, url, params):
         self._num_queries += 1
 
-        return requests.get(url, params=params).json()
+        # Visit main website to get session cookies
+        self.session.get('https://www.ryanair.com/ie/en')
```

Comment:
Excellent, though I would much prefer to only perform this particular `GET` once - either in the constructor, or to cache it on the first call to ` _retryable_query`. My understanding is that this would result in two `GET` requests for every single API call, which seems like too much overhead.

Reply:
Yes, but cookie is expiring after some time (tested) but I don't know for how long. And you need to implement retry solution. I don't have a need to do that. You can develop better solution later on, but for now at least it's working.